ui/pages/pages.scss

```diff
@@ -1,7 +1,7 @@
 /** Please import your files in alphabetical order **/
-@import 'add-token/index';
+@import 'import-token/index';
```

Comment:
Should this be moved down to make this list alphabetical as per the comment above?

Reply:
I'm okay with not enforcing import ordering that isn't yet enforced by our linter

Reply:
Sounds like a challenge. 🥇